src/epcpm/parameterstointerface.py

```diff
@@ -365,20 +366,55 @@ def gen(self):
         else:
             setter_function = parameter.setter_function
 
+        if parameter.constant is None:
+            constant = "NULL"
+            #print("constant is none")
+        else:
+            constant = parameter.constant
+            print("constant is NOT none")
+
```

Comment:
I think this is dead code at this point?  Well, almost.  There's one reference below that could directly reference `parameter.constant`.

```suggestion
```

Reply:
good point.

Reply:
i would have to adjust the code to use parameter.constant instead of constant.

Reply:
Oops, forgot to add that suggestion...